src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et1VettingServiceTest.java

```diff
@@ -199,31 +210,22 @@ void initialBeforeYouStart_TwoRespondentDetails_shouldReturnMarkUp() {
     }
 
     @Test
-    void initialBeforeYouStart_OneAcasNumber_shouldReturnMarkUp() {
-        et1VettingService.initialiseEt1Vetting(caseDetails);
-        String expected = String.format(ACAS_CERT_LIST_DISPLAY, "1234/5678/90");
-        assertThat(caseDetails.getCaseData().getEt1VettingAcasCertListMarkUp())
-                .isEqualTo(expected);
-    }
-
-    @Test
-    void initialBeforeYouStart_NoAcasNumber_shouldReturnMarkUp() {
-        caseDetails = CaseDataBuilder.builder()
-                .withClaimantIndType("Doris", "Johnson")
-                .withClaimantType("232 Petticoat Square", "3 House", null,
-                        "London", "W10 4AG", "United Kingdom")
-                .withRespondentWithAddress("Antonio Vazquez",
-                        "11 Small Street", "22 House", null,
-                        "Manchester", "M12 42R", "United Kingdom",
-                        null)
-                .withRespondentWithAddress("Juan Garcia",
-                        "32 Sweet Street", "14 House", null,
-                        "Manchester", "M11 4ED", "United Kingdom",
-                        null)
-                .buildAsCaseDetails(ENGLANDWALES_CASE_TYPE_ID);
+    void initialBeforeYouStart_returnRespondentAcasDetailsMarkUp() {
         et1VettingService.initialiseEt1Vetting(caseDetails);
-        assertThat(caseDetails.getCaseData().getEt1VettingAcasCertListMarkUp())
-                .isEmpty();
+        String expectedRespondentAcasDetails1 = String.format(RESPONDENT_ACAS_DETAILS, 1, "Antonio Vazquez",
+            "11 Small Street" + BR_WITH_TAB + "22 House" + BR_WITH_TAB + "Manchester" + BR_WITH_TAB + "M12 42R",
+            "1234/5678/90");
+        String expectedRespondentAcasDetails2 = String.format(RESPONDENT_ACAS_DETAILS, 2, "Juan Garcia",
+            "32 Sweet Street" + BR_WITH_TAB + "14 House" + BR_WITH_TAB + "Manchester" + BR_WITH_TAB + "M11 4ED",
+            "2987/6543/01");
+        String expectedRespondentAcasDetails3 = String.format(RESPONDENT_NO_ACAS_DETAILS, 3, "Juan Garcia",
+            "32 Sweet Street" + BR_WITH_TAB + "14 House" + BR_WITH_TAB + "Manchester" + BR_WITH_TAB + "M11 4ED");
```

Comment:
Very minor: Better to illustrate the resulting string rather than appending strings together like this.
Also might be better to have these as constants

Reply:
Yep, changed to use constants instead.